cmd/lncli/walletrpc_active.go

```diff
@@ -604,7 +604,14 @@ var fundPsbtCommand = cli.Command{
 	The optional 'inputs' flag decodes a JSON list of UTXO outpoints as
 	returned by the listunspent command for example:
 
-	    --inputs='["<txid1>:<output-index1>","<txid2>:<output-index2>",...]'
+	    --inputs='["<txid1>:<output-index1>","<txid2>:<output-index2>",...]
+
+	The optional 'change_output' flag permits to choose the scope for the
```

Comment:
We should always talk about "key scope"/"key scopes" in this context as there are other kinds of scopes.